sematic/logging.py

```diff
@@ -24,10 +24,11 @@ def make_log_config(log_to_disk: bool = False):
         },
     }
     if log_to_disk:
+        full_handler_list = ["default", "error"] + stdout_handler_list
         handlers.update(
             {
                 "default": dict(
-                    level="INFO",
+                    level=level,  # type: ignore
```

Comment:
why was this type changed from a string to an int?

Reply:
Because the logs API can accept either, and I wanted to allow passing in `logging.INFO` or `logging.DEBUG` etc. as constants. Those are ints, not strings.